apps/main/src/entities/gauge/model/enabled-conditions.ts

```diff
@@ -0,0 +1,45 @@
+import { checkGaugeValidity } from '@/entities/gauge/lib'
+import { gaugeKeys as keys } from '@/entities/gauge/model'
+import type {
+  AddRewardParams,
+  DepositRewardApproveParams,
+  DepositRewardParams,
+  GaugeQueryParams,
+} from '@/entities/gauge/types'
+import { queryClient } from '@/shared/api/query-client'
+
+export const enabledGaugeStatus = (data: GaugeQueryParams) => {
```

Comment:
This file doesn't add anything to the architecture. It's just boiler plate. Why not inline all functions?